v1/examples/pipeline/steps/custom/harness-approval-step.yaml

```diff
@@ -0,0 +1,31 @@
+version: 1
+kind: pipeline
+spec:
+  stages:
+    - type: approval
+      name: approval
+      id: Approval
+      timeout: 10m
+      when: <+environment.name> != "QA"
```

Comment:
change this to `<+<+environment.name> != "QA">`

Reply:
Done